invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -156,3 +156,72 @@
   display: inline-block;
   margin-right: 0.5em;
 }
+
+/* Styling for the community selection modal */
+
+#community-selection-modal {
+  margin: 0;
+
+  .community-list-results.content {
+    height: calc(55vh - 10em);
```

Comment:
how did you come up with the calculation ?

Reply:
the semantic-ui modal-scrolling class has a max-height of (80vh - 10em) so i took that and made it fixed and smaller so that it does not take up the whole screen. In proportion it works.

Reply:
So i took over the height from the semantic-ui `modal.scrolling` class and instead of a max-height i made it fixed and also take up less of the screen.